src/blueprint/blueprint.rs

```diff
@@ -219,4 +221,53 @@ impl Blueprint {
     // generation of schema cannot fail.
     schema.finish().unwrap()
   }
+
+  /// Generate a dumb schema useful for validation.
+  pub fn to_validation_schema(&self) -> Schema {
+    create(self, Some(SchemaModifiers { no_resolver: true }))
+      .validation_mode(ValidationMode::Strict)
+      .disable_introspection()
+      .finish()
+      .unwrap()
+  }
+
+  pub async fn validate_operation(schema: &Schema, query: &str) -> Vec<Cause<String>> {
+    schema
+      .execute(query)
+      .await
+      .errors
+      .iter()
+      .map(|err| {
+        Cause::new(err.message.clone()).trace(VecDeque::from_iter(
+          err
+            .locations
+            .iter()
+            .map(|loc| format!("line {} column {}", loc.line, loc.column)),
+        ))
+      })
+      .collect()
+  }
+
+  pub async fn validate_operations(&self, operations: Vec<String>) -> Valid<(), String> {
```

Comment:
we need a way to see test output for this without testing it manually. Improving operation tests for this would help